Python/jit.c

```diff
@@ -563,7 +563,7 @@ _PyJIT_Free(_PyExecutorObject *executor)
         executor->jit_side_entry = NULL;
         executor->jit_size = 0;
         if (jit_free(memory, size)) {
-            PyErr_WriteUnraisable(NULL);
+            PyErr_FormatUnraisable("Exception ignored on freeing JIT memory");
```

Comment:
```suggestion
            PyErr_FormatUnraisable("Exception ignored while freeing JIT memory");
```

Reply:
I asked on Discord and I was told to use "when verb+ing" ;-)